### PR TITLE
Handling the error msg from from validation error from djnago serializer

### PR DESCRIPTION
**Pull request by johnyrahul**
## What

- Handling FE exception handling in case of serializer validation errors of the Django framework

## Why

-

## How

- Added else condition if specific backend errors are not set.

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No

## Database Migrations

-  None

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: frontend/src/hooks/useExceptionHandler.jsx [hunk 1]
```diff
@@ -26,6 +26,13 @@ const useExceptionHandler = () => {
           // Handle validation errors
           if (setBackendErrors) {
             setBackendErrors(err?.response?.data);
+          } else {
+            return {
+              title: title,
+              type: "error",
+              content: errors?.[0]?.detail ? errors[0].detail : errMessage,
```
Comment (chandrasekharan-zipstack) [inline frontend/src/hooks/useExceptionHandler.jsx:33]: @johnyrahul why is this required? AFAIK we use a library to format the serializer errors. Can you provide a screenshot of what error we're trying to fix here?

**Review by chandrasekharan-zipstack — APPROVED** [review]

File: frontend/src/hooks/useExceptionHandler.jsx (hunk 1 above)
Comment (johnyrahul) [reply frontend/src/hooks/useExceptionHandler.jsx:33]: @
image

I have discussed this with @tahierhussain . Yes we can improve this to handle all error messages in the list.

**Review by vishnuszipstack — APPROVED** [review]
